.eslintrc.json

```diff
@@ -2,7 +2,7 @@
   "env": {
     "es6": true
   },
-  "extends": ["standard", "prettier"],
```

Comment:
as far as I remember this was needed to detect undefined variables which is a big thing https://github.com/AdExNetwork/adex-market/pull/74/files

Reply:
Yes but in order to add it there are some dependencies that need to be added to the package.json and when I ran the linter there were a lot of errors with this. I can add them and run the linter again.

Reply:
How many is "a lot". As far as I remember, I fixed all errors last time. Please upload the errors here

Reply:
This is after the following dev dependecies were added because are needed for the "standard" option:
"eslint-config-standard": "^14.1.0",
"eslint-plugin-import": "^2.20.1",
"eslint-plugin-promise": "^4.2.1",
"eslint-plugin-standard": "^4.0.1",
-----------------------------------------------------------
/home/rori4/Projects/adex-market/db.js
  10:34  error  ["DB_MONGO_USE_TLS"] is better written in dot notation  dot-notation
  15:32  error  ["DB_MONGO_CA"] is better written in dot notation       dot-notation
  18:16  error  ["DB_MONGO_CERT"] is better written in dot notation     dot-notation
  21:16  error  ["DB_MONGO_KEY"] is better written in dot notation      dot-notation
  23:36  error  ["DB_MONGO_REPLSET"] is better written in dot notation  dot-notation

/home/rori4/Projects/adex-market/helpers/signatureCheck.js
  12:6  error  'usersig' is never reassigned. Use 'const' instead  prefer-const

/home/rori4/Projects/adex-market/helpers/web3/utils.js
  5:7  error  'SignatureModes' is never reassigned. Use 'const' instead  prefer-const

/home/rori4/Projects/adex-market/routes/adSlots.js
  36:9   error  ["$or"] is better written in dot notation   dot-notation
  57:26  error  ["id"] is better written in dot notation    dot-notation
  83:11  error  ["ipfs"] is better written in dot notation  dot-notation

/home/rori4/Projects/adex-market/routes/adUnits.js
  37:9   error  ["$or"] is better written in dot notation   dot-notation
  59:26  error  ["id"] is better written in dot notation    dot-notation
  83:11  error  ["ipfs"] is better written in dot notation  dot-notation

/home/rori4/Projects/adex-market/routes/campaigns.js
  39:24  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  43:12  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  44:13  error  ["depositAsset"] is better written in dot notation                         dot-notation
  47:12  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  51:12  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  77:12  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins

/home/rori4/Projects/adex-market/status-loop/getChannels.js
  18:27  error  ["id"] is better written in dot notation  dot-notation
  18:44  error  ["id"] is better written in dot notation  dot-notation

/home/rori4/Projects/adex-market/test/integration/integration.js
  109:16  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  110:16  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  112:12  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  116:12  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  119:16  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  121:17  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  134:19  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  206:13  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  218:16  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  219:16  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  220:16  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  221:16  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  283:16  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  300:13  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  301:13  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  303:9   error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  307:9   error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  311:9   error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  315:9   error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  319:9   error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  371:27  error  ["Active"] is better written in dot notation                               dot-notation
  478:18  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  480:14  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  483:18  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  484:18  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  496:18  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  498:14  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  502:14  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  506:14  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  509:18  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  510:18  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  529:15  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  693:15  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  750:13  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  803:22  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  807:29  error  ["_id"] is better written in dot notation                                  dot-notation
  853:36  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  857:8   error  'isSorted' is never reassigned. Use 'const' instead                        prefer-const
  886:36  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  912:36  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins

/home/rori4/Projects/adex-market/test/validatorTestMessages.js
  41:25  error  ["timestamp"] is better written in dot notation    dot-notation
  44:25  error  ["lastEvAggr"] is better written in dot notation   dot-notation
  45:25  error  ["isHealthy"] is better written in dot notation    dot-notation
  78:12  error  ["balanceTree"] is better written in dot notation  dot-notation
  81:21  error  ["validUntil"] is better written in dot notation   dot-notation

✖ 66 problems (66 errors, 0 warnings)
  24 errors and 0 warnings potentially fixable with the `--fix` option.